addon/service.js

```diff
@@ -20,8 +20,15 @@ export default Parent.extend(Ember.Evented, {
   t: function(key, data = {}) {
     const locale = this.get('_locale');
     Ember.assert("I18n: Cannot translate when locale is null", locale);
+    const count = get(data, 'count');
 
-    const template = locale.getCompiledTemplate(key, get(data, 'count'));
+    let defaults = get(data, 'default') || [];
+    if (!isArray(defaults)) {
+      defaults = [defaults];
+    }
```

Comment:
This could be `Ember.makeArray(get(data, 'default'))`.
```js
Ember.makeArray(undefined); // []
Ember.makeArray(null); // []
Ember.makeArray('foo'); // [ 'foo' ]
Ember.makeArray([ 'foo', 'bar' ]); // [ 'foo', 'bar' ]
```